### ISSUE TITLE
NVDA is not announcing instruction as "Minimum zero, maximum hundred” at 'Basic SpinButton' spin button.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13209]
<img width="960" alt="Spin button information is not available" src="https://user-images.githubusercontent.com/93735775/147784908-7704e4c9-2bd2-40f6-863e-62196d92dc5a.png">

### Steps to reproduce:
1.Start NVDA Screen Reader.
2.Open link ‘Home - Fluent UI (microsoft.com)’ in edge browser.
3.Tab till ‘Basic SpinButton’ spin button in opened page.
4.Verify whether NVDA is announcing the information about the spin range or not.

### Actual behavior:
When navigated to a spin button, NVDA is announcing the label name and control type, but not announcing the additional instruction about the range of spin and current value.

### Expected behavior:
NVDA should announce about range of spin defined to a spin button such as minimum and maximum values along with current value and other control details.

### System configuration:
Browser: Microsoft Edge Version 96.0.1054.62 (Official build) (64-bit)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA Version 2021.3

#### Windows version:
Windows 11 Version 21H2 (OS Build 22000.376)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, NVDA Version 2021.3.1

**Comment by feerrenrut** [comment]
Related to #13195

**Comment by AnKushSingh05** [comment]
@msftedad is there any update on this bug?.

**Comment by Adriani90** [comment]
This is in general an issue with html elements that have min and max values. Closing as duplicate of #16678 which contains technical info on how to start implementing support for this.
